src/main/java/com/hongliangjie/fugue/io/DataReader.java

```diff
@@ -22,12 +22,12 @@
 
     private static final Logger LOGGER = LogManager.getLogger("FUGUE-TOPICMODELING");
 
-    public DataReader() {
-    }
-
     public Message Read(Message m) throws IOException {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not start with capital letters](https://www.codacy.com/app/hongliangjie/Fugue/file/2178366257/issues/source?bid=3162994&fileBranchId=3163615#l25)